app.py

```diff
@@ -7,15 +7,25 @@
     app.run()
 
 
+@app.route('/test', methods=['POST'])
+def test_path2():
+    return True
+
+
+@app.route('/test')
+def test_path():
+    return "test path"
+
+
 @app.route('/')
 def hello_world():
     conn = sqlite3.connect('data_base.db')
     curs = conn.cursor()
     curs.execute('''CREATE TABLE IF NOT EXISTS users (
-                    Id INTEGER PRIMARY KEY AUTOINCREMENT,
-                    Username TEXT NOT NULL UNIQUE,
-                    Password TEXT NOT NULL
-                    )''')
+                       Id INTEGER PRIMARY KEY AUTOINCREMENT,
+                       Username TEXT NOT NULL,
```

Comment:
Username should be UNIQUE. Why did you remove it?

Reply:
usernames ARE unique, as you can see we check for duplicates in 'check_valid_username' method

Reply:
Did you fix it as we discussed?